### ISSUE TITLE
Early Stopping Integration

### ISSUE THREAD

**Issue by marsbars33** [DAA233/generative-inpainting-pytorch#21]
First off, thank you for providing this implementation! I am new to pytorch and ML in general and I'm about to train on my own dataset, but I wanted to ask if there's a simple way to integrate early stopping? Greatly appreciate any advice you can provide. Thank you in advance :)

**Comment by DAA233** [comment]
Maybe you can watch the loss curve and decide when to early stop.

**Comment by marsbars33** [comment]
Thank you, I will look into this.
